CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -539,6 +539,9 @@ protected virtual async Task StartPackageInstallationInner()
                             deleteList.MoveTo(Path.Combine(_gamePath, $"deletefiles_{Path.GetFileNameWithoutExtension(asset.PathOutput)}.txt"), true);
                         }
 
+                        // Make sure that the Stream is getting disposed first
+                        stream?.Dispose();
```

Comment:
Since the stream is disposed earlier on, we can delete the disposer on L555

Reply:
It should be retained as an exception may occur before L543.

Reply:
Sorry I didn't think of that :derp: